Week1/homework/JS-Exercises/Exercise-10-Compare arrays.js

```diff
@@ -0,0 +1,14 @@
+let vegetables = ["Aubergine", "Broccoli", "Cucumber", "Potato"];
+    fruits   = ["Apple", "Bananas", "Orange", "Kiwi", "Mango", "Watermelon", "Cherries"];
+
+console.log("The length of vegetables is " + vegetables.length + " items.");
+console.log("The length of fruits is " + fruits.length + " items.");
+
+if(vegetables == fruits)
```

Comment:
1. A slight indentation problem here. A well written code not only has a nice logic but is also easy to read which makes code indentation quite important as a programmer
2. Use of triple equals instead of double equals inside `if` block required please